gui/unit-info-viewer.lua

```diff
@@ -451,6 +481,7 @@ function UnitInfo:refresh(unit, width)
     add_chunk(chunks, get_ghostly_chunk(unit), width)
     add_chunk(chunks, get_dead_chunk(unit), width)
     add_chunk(chunks, get_body_chunk(unit), width)
+    add_chunk(chunks, get_average_size(unit), width)
```

Comment:
maybe avoid adding this chunk if the race is dwarf, elephant, or cat. We already know the unit is average/small/large for its race; we don't need to know that the elephant weighs about as much as 1.0 elephants.